src/components/ProductItem.jsx

```diff
@@ -7,14 +7,22 @@ const ProductItem = (props) =>{
   const { name, images, price, categories } = props
 
   return (
-    <Card>
+    <Card data-testid="product-card">
       <ImageContainer>
-        <Image src={(images && images[0]) || undefined} alt={ null }/>
+        <Image src={(images && images[0]) || undefined} alt={ null }
+           data-testid="product-image"
+        />
       </ImageContainer>
-        <Title>{name || "Not Available"}</Title>
-        <Paragraph $type="category">{(categories && categories[0]) || "Generic"}</Paragraph>
-        <Paragraph $type="price">${price || "Not Available"}</Paragraph>
-        <Button>Add to cart</Button>
+        <Title data-testid="product-name">{name || "Not Available"}</Title>
+        <Paragraph $type="category"
```

Comment:
I see that you're using the prop $type for the styles, do you think there's maybe a more suitable name for it?